book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals_list
```

Comment:
- Please move `rentals` to `attr_accessor`. Refer to the comments I made in Rental file.

Reply:
:heavy_check_mark:  rentals add to attr_accessor